contracts/cw3-fixed-multisig/Cargo.toml

```diff
@@ -20,9 +20,9 @@ library = []
 [dependencies]
 cosmwasm-std = { version = "0.10.1", features = ["iterator"] }
 cosmwasm-storage = { version = "0.10.1", features = ["iterator"] }
-cw0 = { path = "../../packages/cw0", version = "0.2.2" }
-cw2 = { path = "../../packages/cw2", version = "0.2.2" }
-cw3 = { path = "../../packages/cw3", version = "0.2.2" }
+cw0 = { version = "0.2.2" }
```

Comment:
Ah, interesting way to do this... as long as no one else imports this, we can hold it back to the older package versions. Nice idea!

Reply:
Yeah, that took a while... :-) happy that I made it compile (and then mergeable).